test/test-config.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tags": {
+    "global:foo": "bar"
+  },
+  "environments": {
+    "dev": {
+      "account": "222222222222",
+      "region": "us-east-1",
+      "resources": {
+        "AWS::Lambda::Function": {
```

Comment:
Are there any tests that rely on resource id rather than type?

Reply:
Yep, I just passed in configuration via a variable as opposed to reading from the test files. See [here](https://github.com/awslabs/aws-ddk/blob/248-ddk-config-common-patterns/test/config.test.ts#L51)